### DISCUSSION TITLE
Comments section shown but no disqus

### DISCUSSION THREAD

**Discussion by secnate** [mmistakes/minimal-mistakes#2600] [Q&A] [answered]
## Environment

- Minimal Mistakes version: 4.19.3
- Ruby gem or remote theme version: Ruby Gem
- Jekyll version: I tried to determine what Jekyll version I am working with but the clue that I can provide as to the Jekyll version is that the website was started this week with the latest Jekyll type.
- Git repository URL: https://github.com/secnate/secnate.github.io
- GitHub Pages hosted (if yes provide URL to site): https://secnate.github.io/)
- Operating system: macOS catalina

## Expected behavior

In my _config.yml file, I enabled disqus comments with the following:
`# Commenting functionality
comments:
  provider: "disqus"
  shortname: "https-secnate-github-io"`

And have pages with the layout being single and the comments setting being enabled to true.

I was expecting to see after each blog post the standard disqus widget. Instead, I see a section titled "Comments" and nothing following it:

<img width="1366" alt="Screen Shot 2020-07-11 at 3 23 59 PM" src="https://user-images.githubusercontent.com/33183110/87232119-96bb9380-c38a-11ea-999b-58c6a8227326.png">

This happens on all pages

## Steps to reproduce the behavior
Go to the website [https://secnate.github.io/](https://secnate.github.io/) and open up a post's page and scroll down towards its bottom.

I'd greatly appreciate any and all help in getting a disqus widget like the following working and integrated:
![image](https://user-images.githubusercontent.com/33183110/87232188-03369280-c38b-11ea-99ae-973233d8df7b.png)

Thanks in advance!

**Comment by mmistakes** [comment, marked as answer]
Check your Disqus settings in _config.yml
You didn’t properly indent `shortname` under the provider key.

https://github.com/secnate/secnate.github.io/blob/master/_config.yml#L44
